images/kcp-upgrade/upgrade.sh

```diff
@@ -22,12 +22,20 @@ SCRIPT_DIR="$(
   cd "$(dirname "$0")" >/dev/null
   pwd
 )"
+
 CONFIG="$(dirname "$(dirname "$SCRIPT_DIR")")/config/config.yaml"
 current_kcp_version="$(yq '.version.kcp' "$CONFIG" | sed 's/v//' )"
 
 latest_kcp_version="$(curl -s https://api.github.com/repos/kcp-dev/kcp/releases/latest | yq '.tag_name' | sed 's/v//' )"
 
-if [[ "$current_kcp_version" != "$latest_kcp_version" ]]; then
+if [ "$latest_kcp_version" == "" ] || [ "$current_kcp_version" == "" ]; then
+  printf "[ERROR] Could not retrieve kcp version: current='%s' latest='%s'" "${current_kcp_version:-not found}" "${latest_kcp_version:-not found}" >&2
+  exit 1
+fi
+
+if [ "$current_kcp_version" != "$latest_kcp_version" ]; then
+  printf "\nNew kcp version is found: %s\n" "$latest_kcp_version"
```

Comment:
This line is not necessary, the next line gives all the info we need.

Reply:
done